code/modules/mob/mob.dm

```diff
@@ -1518,7 +1518,11 @@
 	return WEATHER_EXPOSED
 
 ///Apply a proper movespeed modifier based on items we have equipped
-/mob/proc/update_equipment_speed_mods()
+/mob/proc/update_equipment_speed_mods(mob/user)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.species && (H.species.flags & NO_EQUIP_SLOWDOWN))
+			return
```

Comment:
This isn't an acceptable way to do this, you're passing the same instance as src as an argument

You have to implement an override for /mob/living/carbon/human to do this, call parent if it doesn't have that trait, otherwise don't

Reply:
addressed